test/FunctionalTests/Microsoft.OData.Core.Tests/ScenarioTests/UriBuilder/UrlBuilderWithParameterAliasTests.cs

```diff
@@ -394,6 +394,46 @@ public void BuildExpandOrderby_AliasInFunction()
         }
         #endregion
 
+        #region alias in path segment
+        [Fact]
+        public void BuildFullUri_AliasInFilterPathSegment_AliasAsBoolean()

```

Comment:
>BuildFullUri_AliasInFilterPathSegment_AliasAsBoolean [](start = 20, length = 52)

Can you add UriBuilder tests for expression in $filter segment (rather than alias). i.e., [http://gobbledygook/People/$filter(age](http://gobbledygook/People/$filter(age) ge 18)

Reply:
:+1: